tedana/workflows/tedana.py

```diff
@@ -27,7 +27,9 @@ def _get_parser():
     parser.parse_args() : argparse dict
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument('-d',
+    optional = parser._action_groups.pop()
```

Comment:
Could you maybe add a comment with the link to the SO answer so people know why we are taking this unique approach?

Reply:
Absolutely. Done